### PR TITLE
Add support for releasing Docker images

### PR DESCRIPTION
**Pull request by mongrelion**
This addresses the issue #3 where we want to push to Docker image to the Docker Hub once Travis is done running the tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,5 +15,15 @@ before_install:
   - make test-image deps
   - sleep 30
```
Comment (alex-glv) [inline .travis.yml]: why is this needed?

**Comment by alex-glv** [comment]
Could you add this project to travis so it builds the pr?

File: .travis.yml (hunk 1 above)
Comment (mongrelion) [inline .travis.yml]: I added this when I was trying to fix the end-to-end tests but forgot to remove when I cleaned up.

**Comment by mongrelion** [comment]
@alex-glv you mean this integration?

9tsat
